apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/FeatureFlagsSettings/index.tsx

```diff
@@ -117,17 +170,43 @@ export const FeatureFlagContent = withV3StyleProvider((props: { visible?: boolea
             clearable
           />
           <Alert type="info" title={addFlagHint} showIcon={false} />
+          <Flex flexDirection="row" justifyContent="space-between" mt={5}>
+            <Text>{t("settings.developer.showButtonDesc")}</Text>
+            <Switch
+              name="button-feature-flags-visibible"
+              checked={featureFlagsButtonVisible}
+              onChange={setFeatureFlagButtonVisible}
+            />
+          </Flex>
+          <Button
+            mt={3}
+            variant="color"
+            onClick={resetFeatures}
+            disabled={!hasLocallyOverriddenFlags}
+          >
```

Comment:
the button is going to take all the width possible, creating a giant button ahah maybe make it just fit the text, and center it